frontend/views/containers/payments/ThankYouNoteModal.vue

```diff
@@ -0,0 +1,73 @@
+<template lang="pug">
+modal-template(
+  :a11yTitle='L("View Thank You Modal")'
+  ref='modal'
+)
+  template(slot='title')
+    i18n Thank you note!
+
+  .c-content
+    .c-svg-container
+      svg-hello.c-svg
+
+    .c-note-container
+      i18n.has-text-1.c-label(
+        data-test='memoLabel'
+        :args='{ name: from }'
+      ) {name} Note:
+
+      .c-note.has-text-bold(data-test='memo') {{ thankYouNote }}
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import ModalTemplate from '@components/modal/ModalTemplate.vue'
+import SvgHello from '@svgs/hello.svg'
+
+export default ({
+  name: 'ThankYouNoteModal',
+  components: {
+    ModalTemplate,
+    SvgHello
+  },
+  computed: {
+    ...mapGetters([
+      'groupThankYousFrom'
+    ]),
+    from () {
+      return this.$route.query.from || ''
+    },
+    thankYouNote () {
+      const to = this.$route.query.to || ''
+      return (this.groupThankYousFrom[this.from] || {})[to]
```

Comment:
`this.groupThankYousFrom` always return object. So I think the following is enough.


```
return this.groupThankYousFrom[this.from][to]
```